### PR TITLE
League stat2

### PR DESCRIPTION
**Pull request by z-fitch**
The will add My league stat methods.

### CODE REVIEW

**Review by MWoodshop — APPROVED** [review]
These methods look good. I reviewed them in another PR and left some suggestions / comments but overall this will work as intended.

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -57,13 +57,82 @@ def percentage_visitor_wins
...
+        ids = game.season_id
+        only_ids << ids
+      end
+    uniq_ids = only_ids.uniq
+    count_games = []
+    final_count = Hash.new
+      uniq_ids.each do |id|
+        count_games = @games.find_all {|game|game.season_id.include?(id)}
+        final_count[id] = count_games.length
+      end
```
Comment (awiley33) [inline lib/stat_tracker.rb:81]: I like your logic flow in these methods! I really enjoyed our pairing session when we figured these out together.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -57,13 +57,82 @@ def percentage_visitor_wins
...
+      goal_array = game_team_goals[game_team.team_id.to_sym]
+        if game_team.home_or_away == "home"
+          goal_array << game_team.goals.to_i
+        end
+    end
+    game_team_goals.map do |team, goal_array|
+      game_team_goals[team] = (goal_array.sum.to_f / goal_array.length.to_f)
+    end
+    min_key_value = game_team_goals.min_by{ |team, goals| goals }
+    team = @teams.find do |team|
```
Comment (awiley33) [inline lib/stat_tracker.rb:128]: If we end up having time, these methods would be nice to refactor given that there's so much shared logic with the best/worst offense methods.

**Review by awiley33 — APPROVED** [review]
Great work, Zanna! I left a couple comments.

**Pull request merged**